.gitlab-ci-check-golang-lint.yml

```diff
@@ -0,0 +1,45 @@
+# .gitlab-ci-check-golang-lint.yml
+#
+# This gitlab-ci template performs all static checks from golangci-lint,
+# see https://github.com/golangci/golangci-lint
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-golang-lint.yml'
+#
+# The standard .golangci.yml from this repository is used unless the
+# project including this template has its own version in the root dir.
+#
+# If the repo has some compile time dependencies, the template expect to
+# exist a deb-requirements.txt file with the Debian OS required packages
+#
+
+stages:
+  - test
+
+test:static:
+  stage: test
+  needs: []
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  image: golang:1.17
+  before_script:
+    # Install tool
+    - |
+      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | \
+           sh -s -- -b $(go env GOPATH)/bin v1.31.0
+    # Install compile dependencies
+    - if [ -f deb-requirements.txt ]; then
+        apt-get -qq update &&
+        apt install -yq $(cat deb-requirements.txt);
+      fi
```

Comment:
```suggestion
```

Reply:
Wait, the `deb-requirements` part needs to stay there for 3rd party dependencies that might be required for static analysis. For example [in deployments](https://github.com/mendersoftware/deployments/blob/master/deb-requirements.txt).

Reply:
Do you need them for static checks as well? :thinking: 
In any case, I think the `golangci` image is built on top of a Debian image.

Reply:
Yes, we need them. I added them long time ago for it.

Okey, we will soon find out.